hazelcast/src/test/java/com/hazelcast/client/config/YamlClientFailoverConfigBuilderConfigResolutionTest.java

```diff
@@ -198,8 +198,46 @@ public void testResolveFromWorkDir() throws Exception {
     }
 
     @Test
-    public void testResolveFromClasspath() throws Exception {
+    public void testResolveFromWorkDirYmlButNotYaml() throws Exception {
+        helper.givenYmlClientFailoverConfigFileInWorkDir(42);
+
+        ClientFailoverConfig config = new YamlClientFailoverConfigBuilder().build();
+
+        assertEquals(42, config.getTryCount());
+    }
+
+    @Test
+    public void testResolveFromWorkDirYamlAndYaml() throws Exception {
+        helper.givenYamlClientFailoverConfigFileInWorkDir(42);
+        helper.givenYmlClientFailoverConfigFileInWorkDir(42);
```

Comment:
If this is meant to test the precedence of `.yaml` over `.yml`, then this should be a different int.

Reply:
Have updated